src/web/app/views/beambox/DocumentPanel/DocumentPanel.tsx

```diff
@@ -10,12 +10,8 @@ import Modal from 'app/widgets/Modal';
 import OpenBottomBoundaryDrawer from 'app/actions/beambox/open-bottom-boundary-drawer';
 import PreviewModeBackgroundDrawer from 'app/actions/beambox/preview-mode-background-drawer';
 import SwitchControl from 'app/widgets/Switch-Control';
-import { getSVGAsync } from 'helpers/svg-editor-helper';
-
-let svgCanvas;
-let svgEditor;
-getSVGAsync((globalSVG) => { svgCanvas = globalSVG.Canvas; svgEditor = globalSVG.Editor; });
 
+const { svgCanvas, svgEditor } = window;
```

Comment:
These would be null because svgCanvas and svgEditor is not initiated when document panel is imported.